client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -21,4 +24,24 @@ const PluginSite = ( props ) => {
 	return <PluginSiteJetpack { ...props } />;
 };
 
-export default PluginSite;
+export default connect(
+	( state, ownProps ) => {
+		const pluginSlug = get( ownProps, 'plugin.slug' );
+		const siteId = get( ownProps, 'site.ID' );
+
+		if ( includes( [ 'jetpack', 'vaultpress' ], pluginSlug ) && isSiteAutomatedTransfer( state, siteId ) ) {
+			return {
+				...ownProps,
+				...{
```

Comment:
I'm fine flattening this out but I'd rather do that in a follow up PR since this shape is used on `/plugins/:site`